### DISCUSSION TITLE
Show duration of annotation in the "Add annotation" dialog

### DISCUSSION THREAD

**Discussion by danielfaust** [grafana/grafana#46368] [Dashboards, Panels & Visualizations]
More often than not, when I am creating an annotation, I want to add the duration into the description, in order to have a quick way to find out the duration either when hovering it or when looking at the list. For example "Valve open, 1 hour 12 minutes".

The "Add annotation" dialog shows the starting date and time of the annotation, but in the case of an annotation which spans a time range, it would be helpful if it also showed the 0d h:mm of the duration, for example "1 day 3 h 21 min" or "14 h 5 min" or "9 min".

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
